x-pack/legacy/plugins/code/public/reducers/shortcuts.ts

```diff
@@ -29,23 +29,25 @@ const initialState: ShortcutsState = {
   shortcuts: [helpShortcut],
 };
 
-export const shortcuts = handleActions<ShortcutsState, any>(
+type ShortcutPayload = boolean & null & HotKey;
```

Comment:
why do we need to have the `HotKey` in here? is not quite align with the original code.

Reply:
If you want to match all sorts of actions in this reducer, then why not just keep the original code by applying type in each action, instead of after the `handleActions`?